src/App.js

```diff
@@ -1,9 +1,13 @@
 import React from 'react';
```

Comment:
Let's move `app.js` inside the `component` folder.

![image](https://user-images.githubusercontent.com/31897434/90568251-6ba33b80-e1a3-11ea-990d-e73db732b31b.png)